### PR TITLE
SDKs: Simplify consumption

### PR DESCRIPTION
**Pull request by samijaber**
## Description

- Remove need for `isEditing()`
- Remove need for `getBuilderSearchParams`
  - automatically handle builder preview/editing params in SDKs
- Fix Vue issue not rendering anything
- Switch SvelteKit example for Svelte + Vite

### CODE REVIEW

File: packages/sdks/src/functions/get-builder-search-params/index.ts [hunk 1]
```diff
@@ -20,3 +24,18 @@ export const getBuilderSearchParams = (options: Record<string, string>) => {
...
   return newOptions;
 };
+
+export const getBuilderSearchParamsFromWindow = () => {
+  if (!isBrowser()) {
+    return {};
+  }
+  const searchParams = new URLSearchParams(window.location.search);
+  return getBuilderSearchParams(convertSearchParamsToQueryObject(searchParams));
+};
```
Comment (samijaber) [inline packages/sdks/src/functions/get-builder-search-params/index.ts:34]: --> add default capture of window search params in `getContent`

File: examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue [hunk 2]
```diff
@@ -65,15 +58,12 @@ export default {
...
-      options: getBuilderSearchParams(
-        convertSearchParamsToQueryObject(new URLSearchParams(window.location.search))
-      ),
       userAttributes: {
         urlPath: window.location.pathname,
       },
     }).then(res => {
       this.content = res;
-      this.canShowContent = this.content || isEditing() || isPreviewing();
+      this.canShowContent = this.content || isPreviewing();
```
Comment (samijaber) [inline examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue:66]: See this file for how the builder integration is simplified for all SDK examples

Comment (samijaber) [reply examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue:66]: cc @steve8708 this is how examples look after the changes we talked about

Comment (steve8708) [reply examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue:66]: 😍

**Pull request merged**